tensorflow/contrib/learn/python/learn/estimators/estimator.py

```diff
@@ -1212,8 +1231,10 @@ def _get_predict_ops(self, features):
     Returns:
       `ModelFnOps` object.
     """
+
+    self._set_infer_mode_feature_signature(features)
```

Comment:
This line now implicitly disallows passing None as one of the elements in the features dict. This used to be allowed. 

That is not wrong per se, but as it is now, the error you get is very obscure. We can disallow that (I cannot think of a good reason to pass None in features instead of not including a key). But can you add an explicit check and raise ValueError if any element of the features dict is None?